### ISSUE TITLE
(1.19.3) With EBE chests have "atrifacts?"

### ISSUE THREAD

**Issue by ImGRUI** [FoundationGames/EnhancedBlockEntities#137]
I'm using Quilt 0.17.11 with QFAPI 5.0.0 beta-6, and this happens:

https://user-images.githubusercontent.com/91282421/222943113-204ba139-c26f-48cc-88d6-51d4e79e04bd.mp4

Without EBE it renders normally.

https://user-images.githubusercontent.com/91282421/222943118-1742d4db-52b9-4fee-a752-033b96fe7fad.mp4

Log is empty.
Similar happens to EnderChests.

**Comment by ImGRUI** [comment]
Video reupload.
With EBE:

https://user-images.githubusercontent.com/91282421/222943217-ca8f2365-84e8-42f9-90ea-e27287b9147c.mp4

Without EBE:

https://user-images.githubusercontent.com/91282421/222943225-5e107360-1e72-4ceb-9990-2c1ecf3a5261.mp4

**Comment by FoundationGames** [comment]
This happens when a block entity is on the edge of a render section in vanilla (and all the time with sodium), needs more investigation

**Comment by shark01d** [comment]
I've got the same bug

**Comment by vlad2305m** [comment]
Yeah, could you maybe keep the moving part rendered for a bit longer after switching to the static baked model?
